packages/salesforcedx-vscode-core/src/commands/projectGenerate.ts

```diff
@@ -44,18 +44,16 @@ export class ProjectTemplateItem implements vscode.QuickPickItem {
   }
 }
 
-export class LibraryForceProjectCreateExecutor extends LibraryBaseTemplateCommand<
-  ProjectNameAndPathAndTemplate
-> {
-  private readonly options: forceProjectCreateOptions;
+export class LibraryProjectGenerateExecutor extends LibraryBaseTemplateCommand<ProjectNameAndPathAndTemplate> {
+  private readonly options: projectGenerateOptions;
 
   public constructor(options = { isProjectWithManifest: false }) {
     super();
     this.options = options;
   }
 
-  public executionName = nls.localize('force_project_create_text');
-  public telemetryName = 'force_project_create';
+  public executionName = nls.localize('project_generate_text');
+  public telemetryName = 'project_generate';
```

Comment:
@CristiCanizales renaming the telemetry key for each of these command will break continuity in the telemetry data. I don't mind changing the name, just raising the consequence of doing so. Maybe add a section in the transition doc regarding this so folks can discuss?

Reply:
Sure! I'll keep it as it is for now and after the discussion I'll create a WI if needed to update all telemetry keys 👍